model/task/task.go

```diff
@@ -412,17 +412,18 @@ func (tr TestResult) GetDisplayTestName() string {
 // generation for other log viewers.
 func (tr TestResult) GetLogURL(viewer evergreen.LogViewer) string {
 	root := evergreen.GetEnvironment().Settings().ApiUrl
+	parsleyURL := evergreen.GetEnvironment().Settings().Ui.ParsleyUrl
 	deprecatedLobsterURLs := []string{"https://logkeeper.mongodb.org", "https://logkeeper2.build.10gen.cc"}
 
 	switch viewer {
 	case evergreen.LogViewerHTML:
+		// Return an empty string for logkeeper urls
 		if tr.URL != "" {
 			for _, url := range deprecatedLobsterURLs {
 				if strings.Contains(tr.URL, url) {
-					return strings.Replace(tr.URL, url, root+"/lobster", 1)
+					return ""
```

Comment:
This is the most notable change in behavior we no longer return a legacy lobster URL for the HTML field for resmoke logs. I made this change in an effort to consolidate the logic between resmoke and non resmoke logs.

Reply:
For resmoke logs, will there just be a `Parlsey` and `Raw` link then?